integrations/cdi/datasource-ucp/src/main/resources/META-INF/helidon/native-image/weld-proxies.json

```diff
@@ -0,0 +1,8 @@
+[
+    {
+        "bean-class": "io.helidon.integrations.datasource.ucp.cdi.UCPBackedDataSourceExtension",
+        "ifaces": [
+            "io.helidon.integrations.datasource.ucp.cdi.UCPBackedDataSourceExtension"
```

Comment:
(No action item, necessarily, but since `UCPBackedDataSourceExtension` is a portable extension and not a bean in a normal scope it surprises me that it would be listed in a file called `weld-proxies.json` (i.e. it is never proxied).)

Reply:
Thanks for the insight. Removed this file.